libcudacxx/include/cuda/std/detail/libcxx/include/__config

```diff
@@ -1024,10 +1024,6 @@ typedef __char32_t char32_t;
 
 #define _LIBCUDACXX_DISABLE_EXTENSION_WARNING
 
-#ifdef __cuda_std__
-# undef __cpp_lib_transparent_operators
-#endif // __cuda_std__
```

Comment:
Why is it not needed now?

Reply:
Because we should never ever tamper with what the STL defines

Reply:
I mean, wasn't  this needed for something before? If so, should it undef cccl version of this macro instead?

Reply:
It was needed, because we were messing with the define from the STL